apps/server/src/modules/server/server.module.ts

```diff
@@ -56,6 +57,7 @@ const serverModules = [
 	AuthenticationApiModule,
 	AuthGuardModule,
 	AuthorizationReferenceApiModule,
+	AuthorizationRulesModule,
```

Comment:
Wenn das hier importiert wird, sollte es in server.app eigentlich nicht mehr notwendig sein.